include/seqan3/alphabet/structure/structured_rna.hpp

```diff
@@ -103,23 +109,32 @@ struct structured_rna :
     //!\brief Equals the char_type of sequence_alphabet_type.
     using char_type = underlying_char_t<sequence_alphabet_type>;
 
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    structured_rna() = default;
+    constexpr structured_rna(structured_rna const &) = default;
+    constexpr structured_rna(structured_rna &&) = default;
+    constexpr structured_rna & operator =(structured_rna const &) = default;
+    constexpr structured_rna & operator =(structured_rna &&) = default;
+    ~structured_rna() = default;
+
+    using base_type::base_type; // Inherit non-default constructors
+
+    using base_type::operator=; // Inherit non-default assignment operators
+
+    //!\copydoc cartesian_composition::cartesian_composition(alphabet_type const alph)
+    SEQAN3_DOXYGEN_ONLY(( constexpr structured_rna(alphabet_type const alph) {} ))
+    //!\copydoc cartesian_composition::cartesian_composition(alphabet_subtype const alph)
+    SEQAN3_DOXYGEN_ONLY(( constexpr structured_rna(alphabet_subtype const alph) {} ))
+    //!\copydoc cartesian_composition::operator=(alphabet_type const alph)
+    SEQAN3_DOXYGEN_ONLY(( constexpr structured_rna & operator=(alphabet_type const alph) {} ))
+    //!\copydoc cartesian_composition::operator=(alphabet_subtype const alph)
+    SEQAN3_DOXYGEN_ONLY(( constexpr structured_rna & operator=(alphabet_subtype const alph) {} ))
```

Comment:
I am not sure "alphabet_type" and "alphabet_subtype" are really understandable. Especially "subtype" sounds like it's one half (e.g. of a composition), but in reality, it's the types that the "alphabet_types" can be constructed from, right?

What about "component_types" for the the types that are actual components and "indirect_component_types" for the other ones? And then maybe add a paragraph to the documentation of cartesian composition that  describes the two terms and illustrates with examples the differences?